documentation/docs/25-build-and-deploy/80-adapter-netlify.md

```diff
@@ -107,8 +107,29 @@ Additionally, you can add your own Netlify functions by creating a directory for
 	directory = "functions"
 ```
 
-## Troubleshooting
-
 ### Accessing the file system
 
-You can't access the file system through methods like `fs.readFileSync` in Serverless/Edge environments. If you need to access files that way, do that during building the app through [prerendering](https://kit.svelte.dev/docs/page-options#prerender). If you have a blog for example and don't want to manage your content through a CMS, then you need to prerender the content (or prerender the endpoint from which you get it) and redeploy your blog everytime you add new content.
+You can [use files in Netlify Serverless Functions](https://www.netlify.com/blog/2021/08/12/how-to-include-files-in-netlify-serverless-functions/).
+
+```js
+// @errors: 2307 7031
+/// file: +server.js
+import fs from "node:fs";
+import path from "node:path";
+import { dev } from '$app/environment';
+
+// importing a static asset will return the resolved path in the production build
+import PalatinoBoldFont from "$lib/fonts/PalatinoBold.ttf";
+
+const cwd = process.cwd();
+
+// server assets live in `.netlify/server` when deployed to Netlify
+const dir = dev ? cwd : path.join(cwd, '.netlify/server');
```

Comment:
This feels very hacky. Would love it if we could find a neater solution

Reply:
This, for Netlify is approximately the same as the official documented Vercel solution - if you use next you can yomp the cwd and a magical '/../' path together, but if you don't, you have to use __dirname + string concatenation. Neither work on kit, so I went back to join(process.cwd(), /foo/bar/baz).

and locally of course it's all completely different anyway, since these are built-time post-adapter generated dirs

Reply:
I think I'll revert this documentation change until there's a better solution for Netlify.